tasks/omnibus.py

```diff
@@ -266,9 +266,7 @@ def build(
                 cache_state = None
                 cache_key = omnibus_compute_cache_key(ctx)
                 git_cache_url = f"s3://{os.environ['S3_OMNIBUS_CACHE_BUCKET']}/builds/{cache_key}/{remote_cache_name}"
-                bundle_path = (
-                    "/tmp/omnibus-git-cache-bundle" if sys.platform != 'win32' else "C:\\TEMP\\omnibus-git-cache-bundle"
-                )
+                bundle_path = "/tmp/omnibus-git-cache-bundle"
```

Comment:
it works on msys, but indeed it will probably point to 'C:\msys2\tmp`

Which I *hope* behaves as a `tmpfs` but indeed I can't swear that it does.

Would `%TMPDIR%` be valid in this context?

Reply:
As mentioned in slack, it might be worth to let the Python stdlib take care of this for us to avoid having to guess or write OS-specific code: https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory or https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile could fit the bill here.